content/post/Android_Root101.en.md

```diff
@@ -49,16 +50,18 @@ Place this file inside the folder created after extraction.
 > `cd..` is used to go down one directory
 
 
-Finally pick your phone up and go to the fastboot mode, two methods have been listed below use any:
+Finally, enter fastboot mode on your phone. How-to methods have been listed below(Use any one of the following):
 
-    1. Reboot to Bootloader ->(Hold Power + Volume down)
+1. Reboot to Bootloader ->(Hold Power + Volume down)    {*_For Xiaomi Devices_*}
 
-    2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings
+2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings
```

Comment:
> ADB debugging mode in the developer settings

Have you mentioned how to do that?

Reply:
No I haven't. Please suggest a commit

Reply:
well, I don't know how to do that, 
you tell me

Reply:
also, a suggestion won't suffice imo
you need to make a point 3 for that